miete/templatetags/expr.py

```diff
@@ -0,0 +1,43 @@
+from django import template
+from django.utils.translation import gettext_lazy as _
```

Comment:
wofür ist das? sieht mir nach einer unnötigen sicherheitslücke aus. eval is evil!